#!/bin/sh /usr/share/dpatch/dpatch-run
## 01-autobuild.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Disable modules check to make it buildable without X11.

@DPATCH@

diff -Naurp openerp-client.orig/setup.py openerp-client/setup.py
--- openerp-client.orig/setup.py	2008-11-03 20:53:39.000000000 +0000
+++ openerp-client/setup.py	2008-11-09 16:37:16.000000000 +0000
@@ -122,7 +122,7 @@ def translations():
         trans.append((dest % (lang, name), po))
     return trans
 
-check_modules()
+#check_modules()
 
 # create startup script
 start_script = \
